src/detect-bytejump.c

```diff
@@ -250,9 +259,9 @@ int DetectBytejumpDoMatch(DetectEngineThreadCtx *det_ctx, const Signature *s,
 #endif /* DEBUG */
 
     /* Adjust the detection context to the jump location. */
-    det_ctx->buffer_offset = val;
```

Comment:
Would a debug validation statement here making sure that `jumpptr - payload` is never negative make sense?
Asking bc it seems to me  that `buffer_offset` is an unsigned var.

Reply:
Good point; i'll add that and create a new PR.